modules/Microsoft.Web/staticSites/config/deploy.bicep

```diff
@@ -0,0 +1,50 @@
+@allowed([
+  'appsettings'
+  'functionappsettings'
+])
+@description('Required. Type of settings to apply.')
+param kind string
+
+@description('Required. App settings.')
+param properties object
+
+@description('Conditional. The name of the parent Static Web App. Required if the template is used in a standalone deployment.')
+param name string
+
+@description('Optional. Enable telemetry via the Customer Usage Attribution ID (GUID).')
+param enableDefaultTelemetry bool = true
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+resource staticSite 'Microsoft.Web/staticSites@2022-03-01' existing = {
+  name: name
+}
+
+resource config 'Microsoft.Web/staticSites/config@2022-03-01' = {
+  #disable-next-line BCP225
```

Comment:
Could you elaborate on this line?

Reply:
Must be from the original PR. If removed Bicep shows a warning: 
![image](https://user-images.githubusercontent.com/5365358/180621265-ebccc2d1-5fa8-4576-a2d2-8663ff3adc40.png)

I'd suggest to keep it

Reply:
This is to circumvent a current limitation of the bicep type inference system.

Reply:
Correct. In a separate PR I'm also adding some context to the Wiki.

Reply:
Can this be resolved?

Reply:
I think so @eriqua ?

Reply:
As done for other places where we are silencing false positives, we should add a comment explaining why we are disabling the warning, i.e., why the rule does not apply, as suggested [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/linter#silencing-false-positives)